_lib/cms/components/AvgRating.php

```diff
@@ -22,9 +22,11 @@ public function getFieldSql(): ?string
      */
     public function field(string $fieldName, $value = '', $options = []): string
     {
-        return '<select name="' . $fieldName . '" class="rating" data-section="' . $this->cms->section . '" data-item="' . $this->cms->content['id'] . '"  data-avg="data-avg" ' . $options['attribs'] . '>
-            <option value="">Choose</option>
-            ' . html_options(self::RATING_OPTS, $value, true) . '
-        </select>';
+        $html = [];
+        $html[] = '<select name="' . $fieldName . '" class="rating" data-section="' . $this->cms->getSection() . '" data-item="' . $this->cms->getContent()['id'] . '" data-avg="data-avg" ' . $options['attribs'] . '>';
+        $html[] = '<option value="">Choose</option>';
+        $html[] = html_options(self::RATING_OPTS, $value, true);
+        $html[] = '</select>';
+        return implode('', $html);
```

Comment:
why not concatenate instead of creating an array and imploding?

Reply:
I think it just looks cleaner, but I've updated the code either way